frontend/web/components/App.js

```diff
@@ -275,7 +275,7 @@ const App = class extends Component {
       )
     }
     if (AccountStore.forced2Factor()) {
```

Comment:
I think instead of making all of the isLoginChanges, just adjust this function to return false if 2fa has been set.